api/python/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+[package]
+name = "pyslint"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+description = "Slint Python integration"
+repository.workspace = true
+homepage.workspace = true
+publish = false
+rust-version.workspace = true
+
+[lib]
+name = "slint"
+path = "lib.rs"
+crate-type = ["cdylib"]
+
+[dependencies]
+i-slint-backend-selector = { version = "=1.4.0", path="../../internal/backends/selector" }
+i-slint-backend-testing = { version = "=1.4.0", path="../../internal/backends/testing", optional = true }
+i-slint-renderer-skia = { version = "=1.4.0", path="../../internal/renderers/skia", optional = true, features = ["x11", "wayland"] }
+i-slint-core = { version = "=1.4.0", path="../../internal/core", features = ["ffi"] }
+slint-interpreter = { workspace = true, features = ["default", "display-diagnostics", "internal"] }
```

Comment:
should consider using worksmace dependencies

Reply:
I think that's going to be an issue actually, because the source packages have to be self-contained. Similar to the nodejs packages, we'll need to figure out a way to modify Cargo.toml to use packages from crates.io before building the final package. I guess we can re-use the xtask approach for that. I'll change this to use workspace for now, to ease our own maintenance.

Reply:
`cargo package` might help.

Reply:
That’s a great idea.